hack/run-tests.sh

```diff
@@ -43,11 +43,12 @@ function usage(){
   printf "\n--smoke\n\tFilter to run only the tests tagged with '@smoke'."
   printf "\n--performance\n\tFilter to run only the tests tagged with '@performance'. If not provided and the tag itself is not specified, these tests will be ignored."
   printf "\n--load_factor {INT_VALUE}\n\tSet the tests load factor. Useful for the tests to take into account that the cluster can be overloaded, for example for the calculation of timeouts. Default value is 1."
-  printf "\n--local\n\tSpecify whether you run test in local."
+  printf "\n--local\n\tSpecify whether you run test in local using either a local or remote cluster."
   printf "\n--ci {CI_NAME}\n\tSpecify whether you run test with ci, give also the name of the CI."
   printf "\n--cr_deployment_only\n\tUse this option if you have no CLI to test against. It will use only direct CR deployments."
   printf "\n--load_default_config\n\tTo be used if you want to directly use the default test config contained into ${SCRIPT_DIR}/../test/.default_config"
   printf "\n--container_engine\n\tTo be used if you want to specify engine to interact with images and containers. Default is docker."
+  printf "\n--domain_suffix\n\tTo be used if you want to set a domain suffix for exposed services. Ignored when running tests on Openshift."
```

Comment:
` Ignored when running tests on Openshift.`
Not sure we need to write it here. anyway that domain suffix is automatically into the route

Reply:
For Openshift, this is properly configured. That's this is not necessary to be done when running the tests in Openshift.